### ISSUE TITLE
Reduce frontend Docker image size by serving static page with Nginx

### ISSUE THREAD

**Issue by kongxy10** [equinor/witsml-explorer#924]
**Is your feature request related to a problem? Please describe.**
With the current front-end Docker file, we execute "yarn start", running a node server to serve the application. To serve the containerized application in this way, we need to include the NodeJS runtime and dependent node modules into the docker image, resulting a very large image size (>500 MB).
In fact, serving the application through the Node server is not necessary. Because as of now, the frontend component is built with the "static generation" form.
https://nextjs.org/docs/basic-features/pages#static-generation-recommended
In other words, the application we built through Next js only contains static html, css, scripts, fonts, and other assets.
**Describe the solution you'd like**
In this case, we don't need a node server to host this application. An Nginx to serve the static page is enough. If we switch to the   Nginx base image, the final image size can be drastically reduced to ~23MB
